### DISCUSSION TITLE
Eclipse Finder

### DISCUSSION THREAD

**Discussion by Keatah** [Stellarium/stellarium#2370] [General]
Love the new Eclipse Finder in AstroCalc! Thanks for making it so.

**Comment by worachate001** [comment]
Thank you for the feedback. There are still rooms for improvement, lunar eclipse contact times and map for solar eclipses are enhancement that may be added in the future.

**Reply by Keatah** [reply]
You mean a map showing the path of totality? And percentage of totality?

That’d be oh so super nice.
